src/VirtualClient/VirtualClient.Dependencies/Packaging/WgetPackageInstallation.cs

```diff
@@ -122,6 +122,12 @@ protected override async Task ExecuteAsync(EventContext telemetryContext, Cancel
                         string installationPath = downloadedPackagePath;
                         string packagesDirectory = this.GetPackagePath();
 
+                        // Create packages directory if not present.
+                        if (!this.fileSystem.Directory.Exists(packagesDirectory))
+                        {
+                            this.fileSystem.Directory.CreateDirectory(packagesDirectory).Create();
```

Comment:
Do we need the additional Create()at the end? Or the createDirectory will just do it?

Reply:
yes. Just CreateDirectory() is sufficient.  Changing it.